Add failover_timeout param (#1127)
tarantool/cartridge: rst/topics/failover.rst

in the failover priority list, while instance health is determined according to
the membership status (the SWIM protocol).

> The member is considered to be healthy if both is true:

Reviewer: ```suggestion
The member is considered healthy if both are true:
```

1. It reports either ``ConfiguringRoles`` or ``RolesConfigured`` state;

> 2. And it's SWIM status is either ``alive`` or ``suspect``.

Reviewer: ```suggestion
2. Its SWIM status is either ``alive`` or ``suspect``.
```
